Web-Scraping/amazon_reviews_scraping/spiders/__init__.py

```diff
@@ -0,0 +1,4 @@
+# This package will contain the spiders of your Scrapy project
```

Comment:
Shift these comment to a separate section, better would be in a README file, where you'll keep all your directions and installation guidelines upon this repo!

Reply:
Thanks for the suggestions, on it!